### PR TITLE
operator conventions

### PR DESCRIPTION
**Pull request by prasanthpul**
capture the principles from https://github.com/onnx/onnx/issues/232

### CODE REVIEW

File: docs/OpConventions.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+To maintain consistency in operator signatures, we use the following principles:
+- All attribute names should be lower case and use underscores when it helps with readability
+- Any input/output represented by a single letter is capitalized (i.e. X)
+- Any input/output represented by a full word or multiple words is all lower case and uses underscores when it helps with readability
+- Any input/output representing a bias tensor will utilize the name "B"
+- Any input/output representing a weight tensor will utilize the name “W”
+- “axes” is used when an input, output or attribute is representing multiple axes
+- “axis” is used when an input, output or attribute is representing a single axis
+- Default type for integers will use variants on int, no uints, default is int64
+- Any instance of multiplying a weight tensor by input tensor and adding bias will following the format of W*X^T+B
```
Comment (gramalingam) [inline docs/OpConventions.md]: I don't understand this line (the ^T part). Is this some common notation?

Comment (gramalingam) [reply docs/OpConventions.md]: Other things look good to me.

File: docs/OpConventions.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+To maintain consistency in operator signatures, we use the following principles:
+- All attribute names should be lower case and use underscores when it helps with readability
+- Any input/output represented by a single letter is capitalized (i.e. X)
+- Any input/output represented by a full word or multiple words is all lower case and uses underscores when it helps with readability
+- Any input/output representing a bias tensor will utilize the name "B"
+- Any input/output representing a weight tensor will utilize the name “W”
+- “axes” is used when an input, output or attribute is representing multiple axes
+- “axis” is used when an input, output or attribute is representing a single axis
+- Default type for integers will use variants on int, no uints, default is int64
```
Comment (dzhulgakov) [inline docs/OpConventions.md]: you mean input tensors being int? or IR fields?

File: docs/OpConventions.md (hunk 1 above)
Comment (dzhulgakov) [reply docs/OpConventions.md]: It actually should be `X*W^T+B`. Might make sense to clarify the usual shapes of inputs (e.g. X being `batch X a` and W being `b X a`)

**Pull request merged**